index.bs

```diff
@@ -591,16 +599,16 @@ Registry instances are initially created with the internal slots described in th
 <h4 id="fulfill-fetch" aoid="FulfillFetch">FulfillFetch(loader, entry, payload)</h4>
 
 <emu-alg>
-1. If _entry_.[[Fetch]] is *undefined*, then set _entry_.[[Fetch]] to a new promise.
-1. Fulfill _entry_.[[Fetch]] with _payload_.
+1. If _entry_.[[Fetch]] is *undefined*, then set _entry_.[[Fetch]] to a promise resolved with _payload_.
+1. Else fulfill _entry_.[[Fetch]] with _payload_.
 1. SetStateToMax(_entry_, "translate").
 </emu-alg>
 
 <h4 id="fulfill-translate" aoid="FulfillTranslate">FulfillTranslate(loader, entry, source)</h4>
 
 <emu-alg>
-1. If _entry_.[[Translate]] is *undefined*, then set _entry_.[[Translate]] to a new promise.
-1. Fulfill _entry_.[[Translate]] with _source_.
+1. If _entry_.[[Translate]] is *undefined*, then set _entry_.[[Translate]] to a promise resolved with _source_.
```

Comment:
using al else seems more elegant because there is not need to have a wired promise, it is just a `Promise.resolve()` call.